src/server/mod.rs

```diff
@@ -117,6 +108,32 @@ impl BlockingRequestAction for InitializeRequest {
             ));
         }
 
+        if !dups.is_empty() {
+            let mut msg = String::new();
+            for kv in &dups {
+                msg += &format!("{}: ", kv.0);
```

Comment:
Maybe `write!` would be a better fit here

Reply:
Ok